terraform/Makefile

```diff
@@ -103,7 +103,7 @@ release: build
 endif
 	tar -C $(BUILDDIR) -czf $(RELEASE).tar.gz .
 
-TERRAFORM_EXISTS := $(shell terraform -version 2>/dev/null | grep 'Terraform v1.5')
+TERRAFORM_EXISTS := $(shell terraform -version 2>/dev/null | grep 'Terraform v1.')
```

Comment:
Is this change intentional? Do we want to force a specific TF version or we're OK with everything 1.x?

Reply:
Yes, but I can add it into another PR if you think it's better

I think requiring anything in the `1.x` range is fine, and I was getting annoyed with having to upgrade multiple things